src/routes/share/+page.svelte

```diff
@@ -1,6 +1,14 @@
 <script lang="ts">
     import Navbar from '$lib/components/Navbar.svelte';
-    import { t } from '$lib/data/stores';
+    import { t, language } from '$lib/data/stores';
+    import config from '$lib/data/config';
+    import { base } from '$app/paths';
+
+    //export let language = 'english'; //try switch statement
```

Comment:
Remove dead code

Reply:
If i remove lines 4 and 5, both cause syntax errors since both "base" and "config" are used in the code

Reply:
Actually, it was line 7 which is commented out.

Reply:
Looks like it has been fixed.